recipe/build.sh

```diff
@@ -20,14 +27,23 @@ do
 	    --enable-symlinks \
 	    --enable-termcap \
 	    --enable-pc-files \
-	    --with-termlib \
```

Comment:
Dropping this options gets rid of `libtinfo` and combines it in with `libncurses`.